linux_os/guide/services/sssd/sssd_ssh_known_hosts_timeout.rule

```diff
@@ -5,12 +5,12 @@ prodtype: rhel6,rhel7
 title: 'Configure SSSD to Expire SSH Known Hosts'
 
 description: |-
-    SSSD should be configured to expire keys from known SSH hosts after 1 day.
+    SSSD should be configured to expire keys from known SSH hosts after 5 minutes.
     To configure SSSD to known SSH hosts, set <tt>ssh_known_hosts_timeout</tt>
-    to <tt>86400</tt> under the <tt>[ssh]</tt> section in
+    to <tt>300</tt> under the <tt>[ssh]</tt> section in
     <tt>/etc/sssd/sssd.conf</tt>. For example:
     <pre>[ssh]
-    ssh_known_hosts_timeout = 86400
+    ssh_known_hosts_timeout = 300
```

Comment:
create `var_sssd_ssh_known_hosts_timeout`. See `auditd_audispd_disk_full_action.rule` and `var_audispd_disk_full_action.var` for references.